### PR TITLE
Localization in Staff/* views.

### PR DESCRIPTION
**Pull request by gyakkun**
I try hard to dig out the conflicts between my branch and upstream.

This branch should include no bug fix / enhancement / features but only with localization and translations. Please **check seriously** if any doubts arise.

Some adjustment may feels odd. Please comment.

### CODE REVIEW

File: resources/lang/en/common.php [hunk 1]
```diff
@@ -12,7 +12,8 @@
  */
 
 return [
-    'abbrev-days'          => 'D ',
+    'a-an-art' => '',
```
Comment (HDVinnie) [inline resources/lang/en/common.php]: What is this for?

File: resources/views/Staff/page/create.blade.php [hunk 2]
```diff
@@ -8,34 +8,44 @@
...
     </li>
     <li class="active">
         <a href="{{ route('staff.types.create') }}" itemprop="url" class="l-breadcrumb-item-link">
-            <span itemprop="title" class="l-breadcrumb-item-link-title">Add New Page</span>
+            <span itemprop="title" class="l-breadcrumb-item-link-title">
+                @lang('common.add')
+                @lang(trans_choice('common.a-an-art',false))
+                @lang('common.new-adj')
+                @lang('staff.page')
+            </span>
```
Comment (gyakkun) [inline resources/views/Staff/page/create.blade.php]: `@lang(trans_choice('common.a-an-art',false))` this for example.

File: resources/lang/en/common.php (hunk 1 above)
Comment (gyakkun) [reply resources/lang/en/common.php]: For switch between **a/an** (article word in English). In code `@lang(trans_choice('common.a-an-art',false))` represents "a" and true for "an".

Comment (HDVinnie) [reply resources/lang/en/common.php]: Perfect thanks!

**Comment by HDVinnie** [comment]
@all-contributors please add @gyakkun for translation

**Comment by HDVinnie** [comment]
@gyakkun Do you plan to do more in this Pull Request? Im going to test it locally in a few.

**Comment by gyakkun** [comment]
> @gyakkun Do you plan to do more in this Pull Request? Im going to test it locally in a few.

There's one last page(Staff/warning). Let me finish it.

**Comment by gyakkun** [comment]
I think that's all for this PR. Thank you for reviewing!

**Pull request merged**